src/ir/module-utils.h

```diff
@@ -133,11 +133,10 @@ inline void copyModule(Module& in, Module& out) {
 
 template<typename T>
 inline void renameFunctions(Module& wasm, T& map) {
-  // Update the function itself.
+  // Update the function itself.  In some cases the function might have been
```

Comment:
I'd move this even higher - alongside the `Renaming` comment. This is a "rename all uses, and also the function if it exists" method, and not a simple "rename" method any more, so we should mention it at the toplevel I think.